controllers/storagecluster/cephcluster.go

```diff
@@ -116,6 +117,11 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		return err
 	}
 
+	if r.platform.platform == v1.IBMCloudPlatformType {
```

Comment:
There is a new const being introduced in another IBM PR: https://github.com/openshift/ocs-operator/blob/8c0b66ade6c1187c68cdc8334ee7a9331272b45b/controllers/storagecluster/platform_detection.go#L21 (PR: https://github.com/openshift/ocs-operator/pull/1088)

We should probably wait until that one merges and add the new PlatformType in your if statement.

Reply:
Added.